labs-ios-starterTests/CoreDataTests.swift

```diff
@@ -44,4 +44,17 @@ class CoreDataTests: XCTestCase {
         XCTAssertNotEqual(name, fetchedTopic?.topicName)
     }
 
+    func testCanEstablishQuestionToTopicRelationship() {
+        let fetchedTopic = fetchController.fetchLeaderTopics(with: [1])?[0]
```

Comment:
This looks fine, but for some reason, I think this might have an index out of range error. I'll check this out again tomorrow.

Reply:
Good call, it could probably use some safety